### DISCUSSION TITLE
Shadow  does not update when changing model master branch

### DISCUSSION THREAD

**Discussion by ghost** [google/model-viewer#2784] [Q&A] [answered]
I have been experimenting with the latest changes on the Master branch and I think I have noticed a bug.

### Steps to reproduce:

1. Dynamically change the `src` field on the model-viewer
2. Below in the image I am switching from a couch model to a a light fixture model. The light model has the shadow of the couch model.

Initial model
![image](https://user-images.githubusercontent.com/7875817/133356816-1def59f1-1b24-4f2e-938f-6e984822aefc.png)

Second model
![image](https://user-images.githubusercontent.com/7875817/133356879-527ff06a-9347-4b15-9b2c-9cc7356dc3df.png)

Please let me know if you can't reproduce.

**Note this issue does not occur for me on the latest NPM package. I only see the issue on the latest master branch.**

**Comment by elalish** [comment, marked as answer]
Yep, confirmed; we even have an example to test for this regression, but  I must have failed to check it: https://modelviewer.dev/examples/loading/#cyclingModels

Thanks for testing  master!
